climada_petals/util/config.py

```diff
@@ -24,21 +24,28 @@
 ]
 
 
+import logging
 from pathlib import Path
+import sys
 
 import climada
 import climada.util.config
-from climada.util.config import Config, _fetch_conf, CONFIG_NAME, CONFIG
+from climada.util.config import (
+    Config, _fetch_conf, CONFIG_NAME, CONFIG, remove_handlers, CONSOLE, FORMATTER)
+
+LOGGER = logging.getLogger('climada_petals')
+LOGGER.propagate = False
+remove_handlers(LOGGER)
+LOGGER.addHandler(CONSOLE)
+LOGGER.setLevel(getattr(logging, CONFIG.log_level.str()))
 
 CORE_DIR = Path(climada.__file__).parent
 SOURCE_DIR = Path(__file__).absolute().parent.parent
 
-climada.util.config.CONFIG = Config.from_dict(_fetch_conf([
+CONFIG.__dict__ = Config.from_dict(_fetch_conf([
     CORE_DIR / 'conf',  # default config from the climada repository
     SOURCE_DIR / 'conf',  # default config from the climada_petals repository
     Path.home() / 'climada' / 'conf',  # ~/climada/conf directory
     Path.home() / '.config',  # ~/.config directory
     Path.cwd(),  # current working directory
-], CONFIG_NAME))
-
-climada.CONFIG = climada.util.config.CONFIG
+], CONFIG_NAME)).__dict__
```

Comment:
I know that @emanuel-schmid doesn't like the `__dict__` attribute, but I don't see any alternative in this situation. Furthermore, since the `Config` class is just an "objectified" dictionary, I don't see a problem in handling it like a dictionary.